### PR TITLE
Alert&button

### PR DESCRIPTION
**Pull request by sahiba0915**
Added alert and button component

### CODE REVIEW

File: component/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.avatar-title {
+  font-size: 2rem;
+  padding-top: 1.5rem;
+  color: black;
+}
+.avatar {
+  display: flex;
+  width: 100px;
```
Comment (akash-1318) [inline component/avatar/avatar.css:29]: you should not use pixel unit here

File: component/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.badge-btn {
+  position: relative;
+  display: flex;
+  display: inline;
+  align-items: center;
+  justify-content: center;
+  width: 100px;
+  height: 100px;
```
Comment (akash-1318) [inline component/badge/badge.css:41]: use rem here

**Review by akash-1318 — COMMENTED** [review]
Overall code looks clean from my end.

File: component/avatar/avatar.css (hunk 1 above)
Comment (sahiba0915) [reply component/avatar/avatar.css:29]: done

File: component/badge/badge.css (hunk 2 above)
Comment (sahiba0915) [reply component/badge/badge.css:41]: done

**Comment by poojakelkar** [comment]
Looks good to merge.

**Comment by sahiba0915** [comment]
Merging alert&button branch

**Pull request merged**